src/services/ExposureNotificationService/ExposureNotificationServiceProvider.tsx

```diff
@@ -58,6 +58,7 @@ export const ExposureNotificationServiceProvider = ({
 
   useEffect(() => {
     backgroundScheduler.registerPeriodicTask(async () => {
+      await FilteredMetricsService.sharedInstance().addEvent({type: EventTypeMetric.ActiveUser});
       await exposureNotificationService.updateExposureStatusInBackground();
```

Comment:
Looks like we could move the add event + send inside the updateExposureStatusInBackground function (save some duplication)

Reply:
There is no more sendDailyMetrics here. I put it back at the end of `updateExposureStatusInBackground `. Also the active user event is not publish in `updateExposureStatusInBackground ` anymore.

Reply:
And now the active user is set up to be published right away